recipes/r-sensemakr/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set version = '0.1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sensemakr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/sensemakr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/sensemakr/sensemakr_{{ version }}.tar.gz
+  sha256: 6a1354f05392fa9343b90f69a54022c995651fb3c3d05cb08fa088ef52258caf
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('sensemakr')"           # [not win]
+    - "\"%R%\" -e \"library('sensemakr')\""  # [win]
+
+about:
+  home: https://github.com/carloscinelli/sensemakr
+  license: GPL-2.0-or-later
```

License reads `GPL-3`, so

```suggestion
  license: GPL-3.0-only
```